docs/api-reference/next/image.md

```diff
@@ -111,6 +112,36 @@ Try it out:
 - [Demo the `fill` layout](https://image-component.nextjs.gallery/layout-fill)
 - [Demo background image](https://image-component.nextjs.gallery/background)
 
+### loader
+
+A custom image `loader` to resolve URLs. Defaults to `images` object in `next.config.js`.
```

Comment:
@atcastle is this default correct?

Reply:
Yep, lets link to it (line 72 in image-optimization.md)